techniques/general/ensuring-single-pointer.html

```diff
@@ -19,32 +19,43 @@ <h3>Applicability</h3>
+			<p>The objective of this technique is to provide simple single-click/touch activation alternatives for any functionality offered through more dynamic gestures. Such simple activation involves no requirement for movement after pointer engagement (i.e., after the down-event). This ensures that users who have difficulties performing any form of dragging can still operate all pointer-activated page functions.</p>
 
-			<p class="note">On touch screen devices, author-supplied path-based gestures usually do not work when OS level assistive technologies (AT) like a built-in screenreader are turned on. This is because AT generally consumes a path-based gestures so they would not reach the authored content. For example, a horizontal swipe gesture over the content slider will not work as intended by the author, but move the screen reader focus to the next or previous element. Some gestures may work if the user operates "pass-through gestures" which are often unreliable as they may depend on factors of hardware, operating system, operating system "skin", operating system setting, or user agent.</p>
+		<p>Operations involving dragging, from operating slider controls to free-form drag-and-drop, rely on a user being able to maintain a consistent depressed pointer state while also manipulating the pointer. Users may be significantly challenged in carrying out dragging due to tremors, limitations in dexterity, or technology shortcomings. Offering a means of interaction which does not rely on dragging ensures more users can successfully interact with a page in more circumstances.</p>
+
+
+			<figure>
+				<img src="img/input-range.png" alt="A horizontal line with a round button halfway along the line, labelled volume." />
+				<figcaption>Screenshot of an example slider showing the thumb control in the middle of the track.</figcaption>
+			</figure>
+			
+			<p>Note: Excluded from scope for this technique are alternatives for the cancellation action achieved by a user when moving the pointer from the target area after a down-event has been initiated. This form of cancellation is well-established and the intent of this technique is not to force an alternative method of cancellation, although other methods for Undo may be feasible in many circumstances.</p> 
+		
+      		<p class="note">On touch screen devices, author-supplied dragging gestures usually do not work when OS level assistive technologies (AT) like a built-in screenreader are turned on. This is because AT generally consumes gestures, resulting in them  not reaching the authored content. When custom controls are built on top of native controls, however, these may also be operable with AT gestures such as vertical swiping to change the value (see example 1).</p>
```

Comment:
Rawgit version: https://raw.githack.com/w3c/wcag/0234eff38e359060c448c63d8f838486f42d5e4e/techniques/general/ensuring-single-pointer.html

Reply:
I note that the Technique title implies **all** pointer operations are covered while the description  and the test focus on dragging. The Technique currently does not indicate whether it is a Technique for 2.5.1 Pointer Gestures (this seems to be the case). **IF** it is, I note a mismatch between the scope of 2.5.1 as of the revised understanding text which exempts path-based gestures that do not go through 3 points. The free-form variant of dragging is explicitly included in this Technique. So this Technique seems to be more inclusive that strictly needed to meet 2.5.1 (I think this is a good thing, and may be right on this general level). If the Technique aims to cover all pointer operations, it might usefully include multipoint gestures (split tap, pincer gesture) and list examples for that.
I wonder whether it might be more useful for authors to find Patrick's slider example in a more specific technique focusing on providing single point activation alternatives for control sliders. For a general technique, it may be enough to enumerate several cases where activation is possible with simple taps/clicks: from avoiding path-based /multipoint gestures to providing alternatives such as text input for control sliders, zoom in / out buttons for zoomable maps, alternative single activation controls for content sliders, etc.

Reply:
Yes, intent here is to offer a 'best practices' technique that exceeds the limits of 2.5.1. I previously noted that I'd only tackled dragging so far because the WG had used a lot of cycles regarding dragging. 
> If the Technique aims to cover all pointer operations, it might usefully include multipoint gestures (split tap, pincer gesture) and list examples for that.

Yes, if you have this text somewhere already, please put these multi-point in.  A mapping application may be a good example to put in here (which provides zoom and reposition buttons in addition to pinch and drag). 

> I wonder whether it might be more useful for authors to find Patrick's slider example in a more specific technique focusing on providing single point activation alternatives for control sliders.

I thought that was already appearing somewhere? If you want to have a separate technique for sliders, I think that's fine, although I do find the notion of creating a general technique for a specific interaction a bit problematic, as obviously one could pass slider and still fail the SC on a bunch of different components (i.e., meeting the technique for one component is not going to necessarily allow the whole app to meet the SC).